### ISSUE TITLE
My ruby is too new for twee2?

### ISSUE THREAD

**Issue by alienbogart** [avapoet/twee2#64]
I'm running Manjaro 22.0.0.

```
[~]$ gem install twee2
ERROR:  Error installing twee2:
	twee2-0.5.0 requires Ruby version ~> 2. The current ruby version is 3.0.4.
[~]$ 
```

**Comment by boundingbox** [comment]
Also got this error in Mint 21.2 Cinnamon. 

The directions for installing the specific version called for also no longer functions and returns the following: 
`gpg: directory '/home/user/.gnupg' created
gpg: keybox '/home/user/.gnupg/pubring.kbx' created
gpg: "curl" not a key ID: skipping
gpg: "-sSL" not a key ID: skipping
gpg: "https://get.rvm.io" not a key ID: skipping
gpg: keyserver receive failed: Server indicated a failure`

**Comment by deli73** [comment]
yeah twee2 was made for ruby 2, and it doesn't seem to work with ruby 3 even if you force it to install... likely would require some edits to the code to fix, but this project hasn't been updated in years :/

**Comment by deli73** [comment]
that said, uh, installing ruby 2 with asdf isn't hard, as long as you have the dependencies. so do with that what you will

**Comment by transplier** [comment]
I've made a PR to bring us into modern Ruby. https://github.com/Dan-Q/twee2/pull/65
